modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -1042,4 +980,46 @@ a|The query used a procedure that generated a warning.
 m|WARNING
 |Category
 m|GENERIC
-|===
\ No newline at end of file
+|===
+
+[#_neo_clientnotification_statement_RepeatedRelationshipReference]
+=== Neo.ClientNotification.Statement.RepeatedRelationshipReference
+
+.Notification category details
+[cols="<1s,<4"]
+|===
+|Code
+m|Neo.ClientNotification.Statement.RepeatedRelationshipReference
+|Title
+a|The query returns no results because a relationship variable is bound more than once.
+|Severity
+m|WARNING
+|Category
+m|GENERIC
+|===
+
+.Binding a relationship variable more than once.
+====
+Query::
++
+[source,cypher]
+----
+MATCH ()-[r]->()<-[r]-() RETURN count(*) AS count
+----
+
+Description of the returned code::
+A relationship variable is bound more than once, which leads to no results because relationships must not occur more than once in each result.
+====
+
+.Binding a variable-length relationship variable more than once.
+====
+Query::
++
+[source,cypher]
+----
+MATCH ()-[r*]->()<-[r*]-() RETURN count(*) AS count
+----
+
+Description of the returned code::
+A variable-length relationship variable is bound more than once, which leads to no results because relationships must not occur more than once in each result.
```

Comment:
```suggestion
A relationship is referenced more than once in the query, which leads to no results because relationships must not occur more than once in each result. (Relationship `r` was repeated)
```